src/applications/personalization/dashboard/mocks/server.js

```diff
@@ -3,21 +3,27 @@ const { generateFeatureToggles } = require('./feature-toggles');
 const user = require('./users');
 const { createSuccessPayment } = require('./payment-history');
 const { createAppealsSuccess } = require('./appeals-success');
-const { createDebtsSuccess } = require('./debts');
+const { createDebtsSuccess, createNoDebtsSuccess } = require('./debts');
 const { createClaimsSuccess } = require('./evss-claims');
 const { createHealthCareStatusSuccess } = require('./health-care');
+const { has3Copays } = require('./medical-copays');
 const { v0, v2 } = require('./appointments');
 
+// set to true to simulate a user with debts for /v0/debts endpoint
+const hasDebts = false;
```

Comment:
Non-blocking, but just some thoughts:

I don't know if this is something we want to continue with in the future 🤷🏼  The top section of this server file could get cluttered with these types of little conditionals, but it might be worth thinking about an 'easy' way to configure our mock server responses like this.

I like having this boolean at the top that I can flip and not have to dig into the endpoint list below, but this whole mock server thing is an existing issue, so ill leave it at that and not rant on 😄

Reply:
yeah definitely agreed and noted ✍️